api/src/database/helpers/schema/dialects/oracle.ts

```diff
@@ -40,4 +40,15 @@ export class SchemaHelperOracle extends SchemaHelper {
 
 		return field.type;
 	}
+
+	override async getDatabaseSize(): Promise<number> {
+		try {
+			// const result = await this.knex.raw('select sum(bytes) as "size" from dba_segments');
```

Comment:
I guess that's not required anymore

Reply:
Oops~ 🙈 Removed in https://github.com/directus/directus/pull/22337/commits/502b1b08bc5f07be4701bad8b3eff1b2d6b11f26, thanks!